index.html

```diff
@@ -1,206 +1,405 @@
+        personalized reads; no accounts
+        or sign-ups required.
+      </div>
+      <div class="">
+        <button class="card-tags">html</button>
+        <button class="card-tags">css</button>
+        <button class="card-tags">javascript</button>
+      </div>
+      <div>
+        <button class="card-btn">See Project</button>
+      </div>
+    </div>
+  </div>
+
+
+   <div class="section-2 card desktop-card">
+  <div class="card-details-div">
+    <h2> Uber Navigation </h2>
+    <div class="card-highlights">
+      <p>Uber</p>
+      <img src="images/Counter_1.png">
+      <p> Lead Developer </p>
+      <img src="images/Counter_2.png">
+      <p>2015</p>
+      </ul>
+    </div>
+    <div>
+      A smart assistant to make driving more safe,
+            efficient, and fun by unlocking your most
+            expensive computer: your car.
+    </div>
+    <div class="">
+      <button class="card-tags">html</button>
+      <button class="card-tags"> Ruby on rails</button>
+      <button class="card-tags">css</button>
+      <button class="card-tags">javascript</button>
+    </div>
+    <div>
+      <button class="card-btn">See Project</button>
+    </div>
+  </div>
+  <div class="card-img-div">
+    <img src="images/Portfolio_2.png">
+  </div>
+</div> 
+
+  
+
+  <div class="about-section">
+    <section class="bio-section">
+      <div class="head-1">
+        <h1> About <br>Myself</h1>
+        <p> I’m a software developer! I can help you build a product,
+          feature or website Look through some of my
+          work and experience! If you like what you see and
+          have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a href="#"> LET'S CONNECT</a>
+      </div>
+      <ul class="social-image">
+        <li><a href="https://github.com/david30-maker"><img src="images/github1.png" alt="github logo"></a></li>
+        <li><a href="https://www.linkedin.com/in/david-igbo-464196184"><img src="images/Linkedin_1.png"></a></li>
+         <li><a href="https://venture.angellist.com/v/begin"><img src="images/angellist.png" alt="angel logo"></a></li>
+        <li><a href="https://twitter.com/@okpahdavid"><img src="images/twitter_1.png" alt="Twiiter logo"></a></li>
+        <li><a href="#"><img src="images/medium.png" alt="m logo"></a></li>
+      </ul>
+      <div><button class="card-btn"> Get my resume </button></div>
+    </section>
+
+    <section class="skills-section">
+      <div class="skill">
+        <div id="languages">
+          <h3>Languages</h3>
+          <div id="lo">
+            <div class="language-div">
+              <div>
+                <img src="images/Ellipse-23.png">
+              </div>
+              <div>
+                JavaScript
+              </div>
+            </div>
+            <div class="language-div">
+              <div>
+                <img src="images/Ellipse_22.png">
+              </div>
+              <div>
+                HTML
+              </div>
```

Comment:
- [OPTIONAL] There's one missing underline for frameworks and skill elements. Kindly add it to look like the figma design.

### Your Design
![your-about](https://user-images.githubusercontent.com/26597432/215281932-7d42862e-7130-415c-976f-df8f57a0ccd0.png)

### Figma Design 
![temp-about](https://user-images.githubusercontent.com/26597432/215281941-dd6006ea-8799-4df9-80da-f47164e3ab9d.png)